### DISCUSSION TITLE
Bibtex citation for publications

### DISCUSSION THREAD

**Discussion by jgoldverg** [electricitymaps/electricitymaps-contrib#6678] [General]
Hello all I was wondering if there an official bibtex citation I should use for citing electricity maps.

Thank you!

**Comment by VIKTORVAV99** [comment]
I saw your discussion on slack as well but answering her in case others have the same question.

We don't have a bibTeX file or defined citation but we do have citation examples for our data (somewhat hidden on the dataportal) for APA and MLA.

Here is a screenshot of them:
![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/b41bfde0-dcac-4ce9-bbf3-38e49706046b)

And the current link to the page: https://www.electricitymaps.com/data-portal/australia
(works for any zone in the data portal)

I'll try and make this more prominent on the website and the repo it's somewhat of a low priority on our side.

If the data comes from the API instead you'd have to adjust it a bit since we don't use versioned data in said API.

Hopefully this is of some help at least 🙂
